service/matching/config.go

```diff
@@ -221,6 +223,7 @@ const (
 	loadCauseOtherRead  // any other read-only rpc
 	loadCauseOtherWrite // any other mutating rpc
 	loadCauseForce      // root partition loaded, force load to ensure matching with back logged partitions
+	loadCauseAutoEnable
```

Comment:
this should basically never be an actual load cause since the root should be loaded while any child is loaded. just use loadCauseOtherWrite to keep it simpler?